usr/share/rear/finalize/Linux-i386/670_run_efibootmgr.sh

```diff
@@ -18,61 +20,72 @@ is_true $EFI_STUB && return
 # (cf. https://github.com/rear/rear/pull/2051/files#r258826856):
 test -f "$TARGET_FS_ROOT/$UEFI_BOOTLOADER" || return 0
 
+LogPrint "Creating EFI Boot Manager entries..."
 # Determine where the EFI System Partition (ESP) is mounted in the currently running recovery system:
-esp_mountpoint=$( df -P "$TARGET_FS_ROOT/$UEFI_BOOTLOADER" | tail -1 | awk '{print $6}' )
-# Use TARGET_FS_ROOT/boot/efi as fallback ESP mountpoint:
-test "$esp_mountpoint" || esp_mountpoint="$TARGET_FS_ROOT/boot/efi"
+esp_mountpoint=$( filesystem_name "$TARGET_FS_ROOT/$UEFI_BOOTLOADER" )
+# Use TARGET_FS_ROOT/boot/efi as fallback ESP mountpoint (filesystem_name returns "/"
+# if mountpoint not found otherwise):
+if [ "$esp_mountpoint" = "/" ] ; then
+    esp_mountpoint="$TARGET_FS_ROOT/boot/efi"
+    LogPrint "Mountpoint of $TARGET_FS_ROOT/$UEFI_BOOTLOADER not found, trying $esp_mountpoint"
+fi
 
 # Skip if there is no esp_mountpoint directory (e.g. the fallback ESP mountpoint may not exist).
 # Double quotes are mandatory here because 'test -d' without any (possibly empty) argument results true:
-test -d "$esp_mountpoint" || return 0
-
-BootEfiDev="$( mount | grep "$esp_mountpoint" | awk '{print $1}' )"
-# /dev/sda1 or /dev/mapper/vol34_part2 or /dev/mapper/mpath99p4
-Dev=$( get_device_name $BootEfiDev )
-# 1 (must anyway be a low nr <9)
-ParNr=$( get_partition_number $Dev )
-# /dev/sda or /dev/mapper/vol34_part or /dev/mapper/mpath99p or /dev/mmcblk0p
-Disk=$( echo ${Dev%$ParNr} )
-
-# Strip trailing partition remainders like '_part' or '-part' or 'p'
-# if we have 'mapper' in disk device name:
-if [[ ${Dev/mapper//} != $Dev ]] ; then
-    # we only expect mpath_partX or mpathpX or mpath-partX
-    case $Disk in
-        (*p)     Disk=${Disk%p} ;;
-        (*-part) Disk=${Disk%-part} ;;
-        (*_part) Disk=${Disk%_part} ;;
-        (*)      Log "Unsupported kpartx partition delimiter for $Dev"
-    esac
+if ! test -d "$esp_mountpoint" ; then
+    LogPrint "Skip creating EFI Boot Manager entries (no ESP mountpoint directory $esp_mountpoint)"
```

Comment:
I should use `LogPrintError` instead of `LogPrint` here, and return 1 ?

Reply:
also, reword as "failed to" or "error ..." or something similar instead of just "Skip".

Reply:
Yes, I would much appreciate if you could use `LogPrintError` and `return 1`
plus more explicit wording in the message that something had failed
in this particular case.

Reply:
Done. Also for the other `return` above